src/fights.ts

```diff
@@ -1877,8 +1877,9 @@ const freeKillSources = [
   ),
 ];
 
-export function freeFights(): void {
+export function freeRunFights(): void {
```

Comment:
we run FreeRunFights before regular FreeFights and FreeKillFights. Are you worried about unintended consequences of changing this sequencing?

For example, when we `killRobortCreatures`, we use cartography charges to get penguin access. But that means we can't use cartography when doing extinguisher/xo steals. There are a few other similar things like that. FreeRunFights also benefit from having high buffed +famweight, and after yachtzee we'll lose some of those buffs. And this means we're less likely to have our designer sweatpants charged for a nice day of yachtzeeing (although not _that_ much less likely, given that regular free fights exist)

Reply:
I'm really not too concerned about banderways because it always seems like we either have too many (and garbo does nothing with those extra free runs) or too little (and garbo starts using LTBs for spikolodons). 

In the case where we have too little, we do want to use them over LTBs (although I suppose we should then consider pushing spikolodon forward in the diet schedule).

In the case where we have too many, losing a few banderways due to lost +lbs in most cases literally does not make a difference.

Also, I'd imagine fish heads/penguins are much more valuable than our other uses of cartography (I could be very wrong here), but the old behaviour was to prioritise freeRunFight cartography uses over penguins - if so, wouldn't this new behaviour be better?